installer/dev/console.cpp

```diff
@@ -70,8 +70,8 @@ void WindowsAppRuntimeInstaller::Console::DisplayError(const HRESULT hr)
     }
 
     // Don't log redundant Hr information
-    if (installActivityContext.GetDeploymentErrorExtendedHResult() != S_OK &&
```

Comment:
This doesn't return S_FALSE or other SUCCEEDED cases we want to ignore here?

Reply:
We are now not logging S_FALSE, which indicates success, as this method is named DisplayError(), so I'm _guessing_ that S_FALSE is out of scope. If that's a wrong guess, my apologies, and will happily fix it next time.